src/libs/state.ts

```diff
@@ -91,7 +92,7 @@ export type AuthState = {
 };
 
 export const authStore: Atom<AuthState> = atom<AuthState>({
-  anonymousId: undefined,
+  anonymousId: uuid(),
```

Comment:
does this get stored in localstorage? that seems ideal so that it stays the same unless the anonymous user clears their browser data

Reply:
Yeah, this is cleared on refresh. It is not a cookie but does exist only on the users local app. I think this was the intended behavior before but it was obscured by initializing it in `Metrics.ts`

Reply:
IMO it would be better if this wasnt reset on page refresh, itll give us better numbers, maybe if it is something that is just in local storage and not the auth state?

Reply:
I agree, but I am not sure what I am allowed to store in cookies. I talked with Greg and AppSec about it but I do not think we came to any resolutions.

Reply:
This should possibly be changed instead of reset to the same value after the user signs out.